en/kytos-info.md

```diff
@@ -0,0 +1,21 @@
+# Programmable Network Orchestrator
```

Comment:
Mayara, can you please add a figure here? I suggest you utilize this one: 
https://github.com/kytos-ng/kytos-ng.github.io/blob/master/assets/images/kytosng_logo_color.png